pkg/qemu/qemu.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package qemu is suitable for running QEMU-based integration tests.
+//
+// The environment variable `UROOT_QEMU` overrides the path to QEMU and the
+// first few arguments (defaults to "qemu"). For example, I use:
+//
+//     UROOT_QEMU='qemu-system-x86_64 -L . -m 4096 -enable-kvm'
+//
+// For CI, this environment variable is set in `.circleci/config.yml`.
+package qemu
+
+import (
+	"errors"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/goexpect/goexpect"
```

Comment:
This is blocked on #803. Afterwards, I'll import "github.com/u-root/goexpect" properly and run `go ensure`.